app/src/main/res/values/strings.xml

```diff
@@ -534,14 +534,14 @@
 
   <string-array name="faq_question_answers">
         <item>What image file formats are supported?#####The app supports jpeg, jpg, tiff, gif, psd, bmp, eps, png, etc.</item>
-        <item>Can I create a new text file using the app?#####No, you cannot. You can only convert already created text file to PDF.</item>
+        <item>Can I create a new text file using the app?#####No, you cannot. You can only convert an already created text file to PDF.</item>
```

Comment:
you can create a text file using this app by using "Extract PDF" option.

Reply:
You mean "Extract Text" option?

Reply:
yes.